src/main/kubernetes/artemis/configmaps/artemis-usermanagement-configmap.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+data:
+  _JAVA_OPTIONS: -Xmx512m -Xms256m
+  JHIPSTER_SLEEP: "30"
```

Comment:
Is this actually a string value?

Reply:
I tried it out and applying the kustomization file via kubectl doesn't work if the value is added as a number. So yes, it has to be a string. Same for the other values.